packages/boilerplates/ICanFindOnlineBoilerplatePackages.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Dolittle. All rights reserved.
+*  Licensed under the MIT License. See LICENSE in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { BoilerplatePackageJson } from "./internal";
+
+/**
+ * Responsible for discovering boilerplates
+ *
+ * @export
+ * @interface ICanFindOnlineBoilerplatePackages
+ */
+export interface ICanFindOnlineBoilerplatePackages {
+    
+    /**
+     * Finds boilerplate packages online and and returns the package.json objects of the latest versions of the boilerplates packages compatible with the common tooling major version
+     *
+     * @param {string[]} [keywords] Additional keywords used in search
+     * @param {number} [limit] The limit of boilerplates
+     * @returns {Promise<BoilerplatePackageJson>}
+     * @memberof ICanFindOnlineBoilerplatePackages
+     */
+    discoverLatestOnlineBoilerplates(keywords?: string[], limit?: number): Promise<BoilerplatePackageJson[]>
```

Comment:
A lot of information here in the name of the method. It is in the context of something being capable of finding (or discovering..?? ) boilerplates online. It should be just discover - or find - inconsistency in naming here as well.